real-main/app/handlers/appsync/handlers.py

```diff
@@ -1485,7 +1486,10 @@ def add_chat_message(caller_user, arguments, **kwargs):
 
     chat = chat_manager.get_chat(chat_id)
     if not chat or not chat.is_member(caller_user.id):
-        raise ClientException(f'User `{caller_user.id}` is not a member of chat `{chat_id}`')
+        raise ClientException(
+            f'User `{caller_user.id}` is not a member of chat `{chat_id}`',
+            [UserAuthError.USER_NOT_FOUND],
+        )
```

Comment:
Not sure this error code makes sense. We don't know at this point if the user exists or not.

What do you think about this

```suggestion
if not chat:
    raise ClientException(
        'Chat `{chat.id}` does not exist',
        [ChatErrorCode.CHAT_NOT_FOUND],
    )

if not chat.is_member(caller_user.id):
    raise ClientException(
        'User `{caller_user.id}` is not a member of chat `{chat_id}`',
        [ChatErrorCode.NOT_A_MEMBER_OF_CHAT],
    )
```

As far as https://real-social.atlassian.net/browse/BE-168 goes, I think if the frontend were to display a message 'Chat does not exist' that would be good enough.